pajbot/utils/clean_up_message.py

```diff
@@ -1,7 +1,7 @@
 from typing import Optional
 
 # list of twitch commands permitted, without leading slash or dot
-permitted_commands = {"me"}
+permitted_commands = {"me", "announce"}
```

Comment:
This list is for commands that we don't care if a user uses with user-input - changing a command from a normal privmsg to an action is fine, but using announce is too privileged

```suggestion
permitted_commands = {"me"}
```